src/player.py

```diff
@@ -16,23 +17,42 @@ class Player:
     player_health: int
 
     def __init__(self, input: PlayerInput):
-        self.sprite = arcade.Sprite("assets/vehicle/temp-art.png")
+        self.sprite = arcade.Sprite("assets/vehicle/red-car-top-view.png", 0.5)
+        self.sprite.center_x = 256
+        self.sprite.center_y = 256
         self.input = input
-        self.drive_speed = 100
+        self.drive_speed = 200
         self.turn_speed = 100
         self.primary_weapon = MachineGun(self.input.primary_fire_button, self.sprite)
         self.secondary_weapon = Rocket(self.input.secondary_fire_button, self.sprite)
         self.player_health = 100
 
     def update(self, delta_time):
-        if self.input.x_axis.value < 0:
-            self.sprite.angle -= self.turn_speed * delta_time
-        if self.input.x_axis.value > 0:
-            self.sprite.angle += self.turn_speed * delta_time
-        if self.input.y_axis.value > 0:
-            self.sprite.center_y += self.drive_speed * delta_time
-        if self.input.y_axis.value < 0:
-            self.sprite.center_y -= self.drive_speed * delta_time
+        if self.input.accelerate_button.value > 0:
+            self.sprite.angle -= self.turn_speed * delta_time * self.input.x_axis.value
+            self.sprite.center_x += (
+                self.drive_speed
+                * self.input.accelerate_button.value * math.cos(math.radians(self.sprite.angle))
+                * delta_time
+            )
+            self.sprite.center_y += (
+                self.drive_speed
+                * self.input.accelerate_button.value * math.sin(math.radians(self.sprite.angle))
+                * delta_time
+            )
+        if self.input.brake_button.value > 0:
+            self.sprite.angle += self.turn_speed * delta_time * self.input.x_axis.value
+            self.sprite.center_x -= (
+                self.drive_speed
+                * self.input.brake_button.value * math.cos(math.radians(self.sprite.angle))
```

Comment:
Was the formatter run on this code?  If you're unsure, I'm happy to open the code on my machine and double-check.

I'm marking on this line since it's a bit long, and I don't know what the formatter does for lines like this.

Reply:
I didn't run it, does it run automatically? it's possible I don't have it installed? when i load my twisted-metal repository in vscode i get a popup in the lower right saying "Make not found." is that related?

Reply:
`Make` is unrelated.  I'll double-check the formatting on my end.